shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-scaling/src/test/java/org/apache/shardingsphere/integration/data/pipeline/env/IntegrationTestEnvironment.java

```diff
@@ -175,4 +176,17 @@ public String getActualDataSourcePassword(final DatabaseType databaseType) {
     public static IntegrationTestEnvironment getInstance() {
         return INSTANCE;
     }
+    
+    /**
+     * Just get native database version. 
+     * @param databaseType database type.
+     * @return if matched return version, otherwise return empty.
+     */
+    public List<String> getNativeDatabaseVersions(final String databaseType) {
+        if (StringUtils.equalsIgnoreCase(databaseType, getNativeDatabaseType())) {
+            // Just to occupy the space, will not really use.
+            return Collections.singletonList("keep");
+        }
+        return Collections.emptyList();
+    }
```

Comment:
For `getNativeDatabaseVersions`, it's better to rename it.

And return `Collections.singletonList("keep")` is not readable enough.

Is it possible to return `boolean`?